src/Annotation/AsymmetricKeysGenerator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\pubkey_encrypt\Annotation\AsymmetricKeysGenerator.
+ */
+
+namespace Drupal\pubkey_encrypt\Annotation;
```

Comment:
I modeled all the classes upon the ones provided in Encrypt module. They include the complete namespaces (e.g. `namespace Drupal\encrypt\Annotation;`).

What do you say?